qa/suites/fs/functional/tasks/meta-injection.yaml

```diff
@@ -0,0 +1,5 @@
+tasks:
+- cephfs_test_runner:
+    fail_on_skip: false
+    modules:
+      - tasks.cephfs.test_meta_injection
```

Comment:
the testcase in focus flushes journal and exports the metadata. I hope this isn't prone to some warnings like "slow mds" or "mds behind in trimming"?